packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -138,6 +152,70 @@ const InlineMediaIndicator = styled.div`
       `}
 `;
 
+/*
+ *  Text
```

Comment:
I wouldn't add comments like this. The code is clear enough and comments like this can come with the overhead of ensuring sure they are relevant after code is changed and moved about